.storybook/preview.tsx

```diff
@@ -115,14 +120,18 @@ const withThemeSwitcher: Decorator = (
         return (
             <RenderStateRoot>
                 <ThemeSwitcherContext.Provider value={theme}>
-                    <Story />
+                    <div ref={containerRef}>
```

Comment:
Could you try setting `display: inline-block` on the wrapping div with the ref? I think the Chromatic snapshots were updated because the default `display: block` styling on the div makes it so the story takes up the full width (and that's why there's extra spacing on the right side in stories now, which causes the components to look smaller in the snapshots!)  

Default `block`
![Screenshot 2025-03-20 at 9 58 57 AM](https://github.com/user-attachments/assets/adf873e8-d8f0-424b-a25c-6b6b55aea942)

Trying with `inline-block`
![image](https://github.com/user-attachments/assets/4e6b972e-12a5-4a61-afa0-6c1c1d2acc28)

Reply:
Yes, good catch! I converted it to a span. Does that work?

Reply:
Looks like `span` also takes up the full width! 
![image](https://github.com/user-attachments/assets/811bf45b-0e63-4943-8f68-16812b4d98e0)